packages/astro/src/cli/add/index.ts

```diff
@@ -409,7 +409,9 @@ const toIdent = (name: string) => {
 		// convert to camel case
 		.replace(/[.\-_/]+([a-zA-Z])/g, (_, w) => w.toUpperCase())
 		// drop invalid first characters
-		.replace(/^[^a-zA-Z$_]+/, '');
+		.replace(/^[^a-zA-Z$_]+/, '')
+		// drop version or tag
+		.replace(/@.*$/, '');
```

Comment:
There's a comment above explain some of the transformations being done. Can we also add this `foo@tag` case to the comment above?